platform_channels/lib/src/counter_method_channel.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+/// This class includes implementation of two platform methods [increment],
+/// and [decrement] which are used to increment and decrement value
+/// of count respectively.
+class Counter {
+  /// Creates a [MethodChannel] with the specified name to invoke platform method.
+  /// In order to communicate across platforms, the name of MethodChannel
+  /// should be same on native and dart side.
+  static MethodChannel methodChannel = const MethodChannel('methodChannelDemo');
+
+  /// This method is responsible to increment and return the value of count.
+  static Future<int> increment({int counterValue}) async {
+    try {
+      // Invokes a platform method with given name and argument. The first
```

Comment:
I'm not sure you need this comment block. There's a weird subtly to adding comments to sample code. One the one hand, it can help people figure out what's going on, but on the other it adds length to the overall file and doesn't help those who aren't very accustomed to speaking English.

Typically we tend to add comments that explain functionality specific to the particular app (in other words, nothing that you could just as easily find in the SDK docs). For that reason, I'd recommend cutting this part, since it's mostly just explaining what `invokeMethod` does.

Reply:
Yes, I was also unsure about that. Thanks for pointing out. I'll remember this for next iterations.